src/main/resources/wa-task-initiation-ia-asylum.bpmn

```diff
@@ -42,6 +42,21 @@
       </bpmn:timerEventDefinition>
     </bpmn:intermediateCatchEvent>
     <bpmn:sequenceFlow id="Flow_1azvh3y" sourceRef="createTaskMessage" targetRef="processStartTimer" />
+    <bpmn:subProcess id="Activity_0aq15v4" triggeredByEvent="true">
+      <bpmn:endEvent id="Event_1bcr3ue">
+        <bpmn:incoming>Flow_1th85jg</bpmn:incoming>
+      </bpmn:endEvent>
+      <bpmn:sequenceFlow id="Flow_0icj10p" sourceRef="Event_0piep6v" targetRef="Activity_1etkl8e" />
+      <bpmn:startEvent id="Event_0piep6v" name="Cancel Process" isInterrupting="false">
+        <bpmn:outgoing>Flow_0icj10p</bpmn:outgoing>
+        <bpmn:messageEventDefinition id="MessageEventDefinition_0e8toh3" messageRef="Message_0dksf5o" />
+      </bpmn:startEvent>
+      <bpmn:sequenceFlow id="Flow_1th85jg" sourceRef="Activity_1etkl8e" targetRef="Event_1bcr3ue" />
+      <bpmn:serviceTask id="Activity_1etkl8e" name="Warning Topic" camunda:type="external" camunda:topic="warning-topic">
```

Comment:
maybe we can call this something like wa-warning-topic?  To avoid clashes with another team?